rosdoc2/verbs/build/builder.py

```diff
@@ -47,27 +50,27 @@ def build(self, *, doc_build_folder, output_staging_directory):
         :return: the directory where the documentation was built into, should be
             inside the doc_build_folder. Or None if no artifacts were generated.
         """
-        raise NotImplementedError("Should be implemented by subclasses of Builder.")
+        raise NotImplementedError('Should be implemented by subclasses of Builder.')
 
     def move_file(self, *, source, destination, common_suffix, move=True):
         """
-        Method called on each file as it is moved into the staging directory.
+        Move a file into the staging directory.
 
         May be overridden by downstream builders to selectively skip or rename files.
         """
         if os.path.exists(destination):
             raise RuntimeError(
                 f"Error integrating output from builder '{self.name} ({self.builder_type})': "
                 f"file '{common_suffix}' already exists in destination '{destination}'. "
-                f"This usually occurs when two builders generate the same file in "
-                f"the output directory.")
+                f'This usually occurs when two builders generate the same file in '
+                f'the output directory.')
```

Comment:
@clalancette nit: no need for these to be f-strings, regular strings would do. Same elsewhere for f-strings that lack `{...}` placeholders.

Reply:
Good call, will fix.